### DISCUSSION TITLE
Where is the LDAP settings

### DISCUSSION THREAD

**Discussion by JuanCsrl** [community/community#23542] [Enterprise]
<p>I can’t find the ldap settings to configure. I read the docs but the tutorial screens are different… what’s wrong?<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/8/e837759c70f80b08dca7d678191d0029d5ad9e99.png" data-download-href="/uploads/short-url/x8hAQSadUp5Nam8n7RMYWZ0vbqV.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/32464161/181077821-bac34e71-1363-4b8b-a546-6c892c1d2f29.png" alt="image" data-base62-sha1="x8hAQSadUp5Nam8n7RMYWZ0vbqV" width="690" height="383" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e837759c70f80b08dca7d678191d0029d5ad9e99_2_690x383.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e837759c70f80b08dca7d678191d0029d5ad9e99_2_1035x574.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e837759c70f80b08dca7d678191d0029d5ad9e99_2_1380x766.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e837759c70f80b08dca7d678191d0029d5ad9e99_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1460×812 135 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by byrneh** [comment, marked as answer]
<p>Hi <a class="mention" href="/juancsrl">@JuanCsrl</a> welcome to the community.</p>
<p>The LDAP settings are definetly there on GitHub Enterprise Server and are accessed fron within the mmanagement console. Please note LDAP is only available on GitHub Enterprise Server which is something a company sets up and host themselves.</p>
<p>It is not possible to configure LDAP on the GitHub cloud product offerings hosted on <a href="https://github.com" rel="noopener nofollow ugc">https://github.com</a> just in case that is what you are trying to do?</p>

**Comment by JuanCsrl** [comment]
<p>Tks, helped me a lot</p>
